00-introduction.md

```diff
@@ -0,0 +1,6 @@
+# BOLD #0: Introduction
+
+Your implementation of the OpenDex specifications will be able to participate in the global decentralized trading network! 
+
+Currently OpenDex supports Java, Python, Objective-C, C++, Dart, Go, Ruby, and C#.
```

Comment:
Why? How about Rust? How about... ?

Reply:
good point these were just the ones listed on google's site